### ISSUE TITLE
Missing semicolon

### ISSUE THREAD

**Issue by ducreyna** [driftyco/ionic-service-push-client#7]
Hello,

I'm using your module in an Ionic/Angular project and I have noticed that a semicolon is missing at the end of the file. A Javascript's error occurs and the build fails.

Best regards

```
.factory('$ionicPushActions', [
    '$rootElement',
    '$injector',
function($rootElement, $injector) {
  return {
    run: function(notification) {
      if(notification.$state) {
        // Auto navigate to state
        var injector = $rootElement.injector();
        if(injector.has('$state')) {
          $state = injector.get('$state');
          var p = {};
          try {
            p = JSON.parse(notification.$stateParams);
          } catch(e) {
          }
          $state.go(notification.$state, p);
        }
      }
    }
  }
}]);
```

**Comment by igorlamos** [comment]
+1

**Comment by Fuiste** [comment]
Whoops, should be fixed in v0.1.11
